admin/proc-rhdh-monitoring-logging-aks.adoc

```diff
@@ -0,0 +1,52 @@
+[id='proc-rhdh-monitoring-logging-aks_{context}']
+= Monitoring and logging with Azure Kubernetes Services (AKS) in {product}
+
+Monitoring and logging are integral aspects of managing and maintaining Azure Kubernetes Services (AKS) in {product}. With features like Managed Prometheus Monitoring and Azure Monitor integration, administrators can efficiently monitor resource utilization, diagnose issues, and ensure the reliability of their containerized workloads.
+
+To enable Managed Prometheus Monitoring, use the `-enable-azure-monitor-metrics` option within either the `az aks create` or `az aks update` command, depending on whether you're creating a new cluster or updating an existing one:
+
+[source,bash]
+----
+az aks create/update --resource-group <your-ResourceGroup> --name <your-Cluster> --enable-azure-monitor-metrics
+----
+
+The previous command installs the metrics add-on, which gathers https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/prometheus-metrics-overview[Prometheus metrics]. Using the previous command, you can enable monitoring of Azure resources through both native Azure Monitor metrics and Prometheus metrics. You can also view the results in the portal under *Monitoring -> Insights*. For more information, see https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/monitor-azure-resource[Monitor Azure resources with Azure Monitor].
```

Comment:
_The previous command installs the metrics add-on, which gathers [Prometheus metrics](https://learn.microsoft.com/en-us/azure/azure-monitor/essentials/prometheus-metrics-overview). You can use it to monitor Azure resources..._

Reply:
You can use it <what>? Usage of **it** seems ambiguous. In this sentence, the command itself enables (--enable-azure-monitor-metrics).